src/analyses/natural_loops.h

```diff
@@ -60,6 +76,8 @@ class natural_loops_templatet
   void compute_natural_loop(T, T);
 };
 
+/// Helper class for a concretized version of
```

Comment:
This isn't really a helper class, it's effectively a typedef, so maybe just call it "A concretized version of...